creator-node/src/routes/files.js

```diff
@@ -289,61 +367,70 @@ const getCID = async (req, res) => {
         time: `${Date.now() - startMs}ms`
       })
     }
-  } else {
-    // Check if CID record is in DB, error if not
-    startMs = Date.now()
-    try {
-      const queryResults = await models.File.findOne({
-        where: {
-          multihash: CID
-        },
-        order: [['clock', 'DESC']]
-      })
+  }
+
+  /**
+   * If not found on FS, check if CID record is in DB, error if not found
+   */
+  startMs = Date.now()
+  try {
+    const queryResults = await models.File.findOne({
+      where: {
+        multihash: CID
+      },
+      order: [['clock', 'DESC']]
+    })
+    decisionTree.push({
+      stage: `DB_CID_QUERY`,
+      time: `${Date.now() - startMs}ms`
+    })
+
+    if (!queryResults) {
       decisionTree.push({
-        stage: `DB_CID_QUERY`,
-        time: `${Date.now() - startMs}ms`
+        stage: `DB_CID_QUERY_CID_NOT_FOUND`
       })
-
-      if (!queryResults) {
-        decisionTree.push({
-          stage: `DB_CID_QUERY_CID_NOT_FOUND`
-        })
-        logGetCIDDecisionTree(decisionTree, req)
-        return sendResponse(
-          req,
-          res,
-          errorResponseNotFound(
-            `${logPrefix} No valid file found for provided CID`
-          )
-        )
-      } else if (queryResults.type === 'dir') {
-        decisionTree.push({
-          stage: `DB_CID_QUERY_CONFIRMED_DIR`
-        })
-        logGetCIDDecisionTree(decisionTree, req)
-        return sendResponse(
-          req,
-          res,
-          errorResponseBadRequest('this dag node is a directory')
+      logGetCIDDecisionTree(decisionTree, req)
```

Comment:
im seeing these log commands sprinkled throughout this method. can we aggregate these calls into one log? if we log the decision tree each time we add to it, we're going to be printing repeated steps + add a lot of logs to our content nodes. this route is so heavily visited. I dont think it's a good idea to have all these logs

Reply:
we don't log decision tree each time we add to it - each getCID call will only ever log the tree once
there are multiple calls to log because there are multiple return conditions, so each one needs a preceding log call

Reply:
oh ok, I see the difference here. thanks for clarifying. is there a reason we added a decision tree in this flow?